src/app/_components/FeaturedEcosystemProjects.tsx

```diff
@@ -18,10 +18,10 @@ export function FeaturedEcosystemProjects() {
   }
 
   return (
-    <ul className="gap-8 sm:gap-6 sm:grid sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 list-none">
+    <ul className="list-none gap-8 sm:grid sm:grid-cols-2 sm:gap-6 md:grid-cols-3 lg:grid-cols-4">
       {featuredEcosystemProjects.map(({ slug, title, description }) => (
         <li key={slug} className="ml-0">
-          <Badge featured />
+          <Badge variant="primary">Featured Project</Badge>
```

Comment:
If we set the default to `variant = 'primary', in the component, we won't have to pass it always.